src/mysql/result_set.cr

```diff
@@ -88,6 +88,23 @@ class MySql::ResultSet < DB::ResultSet
     end
   end
 
+  def read(t : UUID.class)
+    row_packet = @row_packet.not_nil!
+    is_nil = @null_bitmap[@column_index + 2]
+    col = @column_index
+    @column_index += 1
+
+    if is_nil
+      nil
+    elsif @columns[col].flags.bits_set?(128)
+      # Check if binary flag is set
+      # https://dev.mysql.com/doc/dev/mysql-server/latest/group__group__cs__column__definition__flags.html#gaf74577f0e38eed5616a090965aeac323
```

Comment:
💚

Reply:
I would have expected to see an enum with these flags, i was thinking about adding them if we will start to use it.